drupal/Drupal.py

```diff
@@ -467,13 +467,18 @@ def environment_indicator(www_root, repo, branch, build, buildtype, alias, site,
 # Function used by Drupal 8 builds to import site config
 @task
 @roles('app_primary')
-def config_import(repo, branch, build, buildtype, site, alias, drupal_version, previous_build, sites_deployed=None):
+def config_import(repo, branch, build, buildtype, site, alias, drupal_version, import_config_method, cimy_mapping, previous_build, sites_deployed=None):
   with settings(warn_only=True):
     # Check to see if this is a Drupal 8 build
     if drupal_version > 7:
+      if import_config_method == "cimy":
```

Comment:
You find yourself running the same routine several times. I would move this to DrupalConfig.py and have a function that returns the command to execute. In fact, I'd call the check_cmi_tools_exists() functions from within it as well. So have a function that sets and checks the command will succeed, then passes it back.